mobile_verifier/src/reward_shares.rs

```diff
@@ -203,52 +204,147 @@ pub fn dc_to_mobile_bones(dc_amount: Decimal, mobile_bone_price: Decimal) -> Dec
         .round_dp_with_strategy(DEFAULT_PREC, RoundingStrategy::ToPositiveInfinity)
 }
 
-#[derive(Default)]
-pub struct RadioShares {
-    radio_shares: HashMap<Option<String>, Decimal>,
+#[derive(Debug)]
+struct RadioPoints {
+    heartbeat_multiplier: Decimal,
+    coverage_object: Uuid,
+    seniority: DateTime<Utc>,
+    points: Decimal,
 }
 
-impl RadioShares {
-    fn total_shares(&self) -> Decimal {
-        self.radio_shares
-            .values()
-            .fold(Decimal::ZERO, |sum, amount| sum + amount)
+impl RadioPoints {
+    fn new(heartbeat_multiplier: Decimal, coverage_object: Uuid, seniority: DateTime<Utc>) -> Self {
+        Self {
+            heartbeat_multiplier,
+            seniority,
+            coverage_object,
+            points: Decimal::ZERO,
```

Comment:
we instantiate this to ZERO but where does this value ever get set to anything else?